source/fundamentals/crud/write-operations/insert.txt

```diff
@@ -67,12 +88,32 @@ inserts the ``List`` using the ``insertMany()`` method:
    :start-after: begin insertManyExample
    :end-before: end insertManyExample
 
-The output of the operation above should look similar to the following:
+Your output should look something like this:
 
 .. code-block:: json
    :copyable: false
 
-    AcknowledgedInsertManyResult{ insertedIds = { 0 = BsonInt32{value=1}, 1 = BsonInt32{value=2} }}
+    IntertedIds: [ BsonInt32{ value=1 }, BsonInt32{ value=2 }]
```

Comment:
1) I think there's a typo here.
2) Could we print normal Java Integer values rather than BsonInt32? That way, the output could be more human-readable, e.g:
"Inserted documents with the following ids: [ 1, 2 ]"